packages/babel-parser/src/plugins/estree.js

```diff
@@ -194,6 +194,33 @@ export default (superClass: Class<Parser>): Class<Parser> =>
       }
     }
 
+    parseMaybePrivateName(...args: [boolean]): any {
+      const node = super.parseMaybePrivateName(...args);
+      if (node.type === "PrivateName") {
+        return this.convertPrivateNameToPrivateIdentifier(node);
+      }
+      return node;
+    }
+
+    convertPrivateNameToPrivateIdentifier(
+      node: N.PrivateName,
+    ): N.EstreePrivateIdentifier {
+      const name = super.getPrivateNameSV(node);
+      node = (node: any);
+      delete node.id;
+      node.name = name;
+      node.type = "PrivateIdentifier";
+      return node;
+    }
+
+    isPrivateName(node: N.Node): boolean {
+      return node.type === "PrivateIdentifier";
+    }
+
+    getPrivateNameSV(node: N.Node): string {
```

Comment:
What does `SV` mean?

Reply:
It has been copied from the spec: https://tc39.es/ecma262/#sec-static-semantics-sv
It means "string value".

Reply:
Thank you!

Reply:
It is short for "StringValue". But `getPrivateNameStringValue` is a bit too long, do you have recommendation? (Naming is hard).